RELEASE-NOTES.txt

```diff
@@ -4,6 +4,7 @@
 
 18.0
 -----
+* [*] Fixed a bug that would make it impossible to scroll the plugins the first time the plugin section was opened.
```

Comment:
Just mentioning/reminding folks that it's too late for additions to 18.0 release notes at this point (past 1st week of beta, release notes already sent to Editorial then translation vendor)